crates/noirc_evaluator/src/ssa_refactor/ir/dfg.rs

```diff
@@ -91,41 +100,54 @@ impl DataFlowGraph {
     }
 
     /// Inserts a new instruction into the DFG.
+    /// This does not add the instruction to the block or populate the instruction's result list
     pub(crate) fn make_instruction(&mut self, instruction_data: Instruction) -> InstructionId {
         let id = self.instructions.insert(instruction_data);
 
         // Create a new vector to store the potential results for the instruction.
         self.results.insert(id, Default::default());
+
         id
     }
 
```

Comment:
can you comment what this does and why we use it?

Reply:
The preview lines on github for your comment are a bit off so I'm assuming you're reference `make_value`. I've added a comment there.